package.json

```diff
@@ -22,9 +21,8 @@
     "dev:compat": "es-dev-server --node-resolve --compatibility esm --open /playground/",
     "lint": "eslint --env browser --fix src/**/*.ts",
     "compile": "rimraf lib && rollup -c",
-    "minify-iife": "terser lib/doofinder.js -c -m --comments /^nothing/ --keep-classnames --keep-fnames -o lib/doofinder.min.js",
```

Comment:
@sonic182 you removed options from terser after enabling the rollup plugin… are those the defaults?

Reply:
keep classnames and fnames yes are default true, the commets stuff is missing

Reply:
ok!